modules/caddyhttp/serverless/testdata/pyechoserver/app.py

```
@@ -0,0 +1,73 @@
+from fastapi import FastAPI, Request, HTTPException
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)